src/main/resources/templates/home.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
I believe this document can be removed, as the feed.html file serves this purpose.